Tropos/Controllers/TRWeatherController.m

```diff
@@ -37,6 +39,7 @@ - (instancetype)init
     self.geocodeController = [TRGeocodeController new];
     self.forecastController = [TRForecastController new];
     self.settingsController = [TRSettingsController new];
+    self.unitSystemChanged = [[self.settingsController unitSystemChanged] replayLastLazily];
```

Comment:
It certainly wouldn't break without it.

Mainly: I'm not aware of any downside to the laziness, and I think deferring the work until it's actually needed is generally a good idea.